File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
                                    
    static String answer;
    static String lt;
    static String rt;
    public String solution(int[] food) {
        lt = "";
        answer="";
        rt="";
        for (int i = 0; i < food.length; i++) {
            lt += Integer.toString(i).repeat((food[i]/2));
            rt += Integer.toString(food.length-i-1).repeat((food[food.length-i-1]/2));
        }
        answer = lt + '0' + rt;
        return answer;
}
}

File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    lt,rt = "",""
    answer = ""
    for i in range(len(food)):
        lt += str(i) * (food[i]//2)
        rt += str((len(food)-i-1))*(food[len(food)-i-1]//2)
    answer = lt+'0'+rt
    return answer